app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/check_all_answers.scala.html

```diff
@@ -82,6 +95,38 @@
 
  @pageHeading(title)
 
+  @if(showSubmissionDetails) {
+      <h2 class="heading-medium">Submission Details</h2>
+      @cyaSection() {
+          @if(!isAgent) {
+              @cyaRow(
+                  messages("CGT reference number"),
```

Comment:
it's good practice to stick these messages in the messages file - having said that, I think we're only going to need to display this page in English. Is there a way to make sure this is always in English when creating the HTML for DMS?